Formula/assh.rb

```diff
@@ -1,9 +1,8 @@
 class Assh < Formula
   desc "Advanced SSH config - Regex, aliases, gateways, includes and dynamic hosts"
   homepage "https://github.com/moul/advanced-ssh-config"
-  url "https://github.com/moul/advanced-ssh-config/archive/v2.8.0.tar.gz"
-  sha256 "17656a6ac562707d6e85df44c1ccd04276fb1c08f1ff6a002291f4cb88880069"
-  revision 1
+  url "https://github.com/moul/advanced-ssh-config/archive/v2.9.1.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/moul/assh/archive/v2.9.1.tar.gz"
```

Also, they have renamed a repo